doc/getting-started/how-to-install-smash.md

```diff
@@ -72,20 +83,26 @@ The most important points of it are here:
 
 70+50+255+5+64 = 444 bytes.
 
-## How to run it with the node
+## How to run SMASH with the Cardano node
+
+First run the Cardano node as a relay, e.g.:
 
-First run the node. The genesis should be generated from Nix. You can run smash with:
 ```
-SMASHPGPASSFILE=./config/pgpass stack run smash-exe -- run-app-with-db-sync --config ../cardano-db-sync/config/testnet-config.yaml --genesis-file /nix/store/pdnsmv1gi6llr92fk7sgs3j2gcphm6fm-shelley-testnet-genesis.json --socket-path ../cardano-node/state-node-shelley_testnet/node.socket --schema-dir schema/
+cardano-node --genesis-file genesis.json --socket-path node.socket --config config.yaml
+```
+
+You can then run ``smash`` using e.g:
+```
+SMASHPGPASSFILE=./config/pgpass stack run smash-exe -- run-app-with-db-sync --config config.yaml --genesis-file genesis.json --socket-path node.socket --schema-dir schema/
 ```
 
 ## How to test this works?
 
-You can run the provided example and try out these commands (presuming you know what CURL is and how to use it).
+You can run the provided example and try out these commands
 ```
 curl --verbose --header "Content-Type: application/json" --request GET http://localhost:3000/api/v1/metadata/ed25519_pk1z2ffur59cq7t806nc9y2g64wa60pg5m6e9cmrhxz9phppaxk5d4sn8nsqg
 
-curl --verbose --user ksaric:test --header "Content-Type: application/json" --request POST --data '{"blacklistPool":"xyz"}' http://localhost:3000/api/v1/blacklist
+curl --verbose --user ksaric:cirask --header "Content-Type: application/json" --request POST --data '{"delistPool":"xyz"}' http://localhost:3000/api/v1/delist
```

Comment:
The user password is `test` so it's compatible with the example insertion in the database.